### ISSUE TITLE
Views lack prominent headers or titles

### ISSUE THREAD

**Issue by ssciolla** [tl-its-umich-edu/my-learning-analytics#801]
Thank you for contributing to this project!

- Make sure to search the [issues](https://github.com/tl-its-umich-edu/my-learning-analytics/issues) for duplicates first!

## Expected behavior (A clear and concise description of what you expected to happen) : 

Each view should have a clear heading or title so that all users will clearly understand when a view or route change has occurred.

## Describe the bug (Tell us what happens instead of the expected behavior) :

From the 10/28/19 accessibility review:

"There is no visible or programmatically discoverable way to tell in what view of this [single page application] the user is currently in. This is important for the blind persona."

## Additional context (Add any other context about the problem here) :

The Resources Accessed and Grades Distribution views have headers, but they are nested within Material UI Paper components that contain the rest of the content. Assignment Planning does not have a header (hopefully this will be addressed as part of work on issue #691 and PR #702). Perhaps headers or titles should live in their own Paper component; creating a new custom component might be appropriate.

The review provides these suggested solutions: "Change the title on view/route change. Or pass the focus to a heading that announces the view change."

**Comment by justin0022** [comment]
I appreciate all of these accessibility issues. It'll take time to implement but well worth the effort.

**Comment by jennlove-um** [comment]
testing passes

**Comment by gsilver** [comment]
@jennlove-um - testing passes indeed!
